ietf-layer0-types-ext.yang

```diff
@@ -51,4 +51,397 @@ module ietf-layer0-types-ext {
    
    The typedefs and groupings will be added here. */
 
-}
+// identity
+
+  identity modulation {
+    description "base identity for modulation type";
+  }
+
+  identity QPSK {
+    base modulation;
+    description
+      "QPSK (Quadrature Phase Shift Keying) modulation";
+  }
+
+  identity DP-QPSK {
+    base modulation;
+    description
+      "DP-QPSK (Dual Polarization Quadrature
+       Phase Shift Keying) modulation";
+  }
+
+  identity QAM8 {
+    base modulation;
+    description
+      "8QAM (8-State Quadrature Amplitude Modulation) modulation";
+  }
+
+  identity QAM16 {
+    base modulation;
+    description
+      "QAM16 (Quadrature Amplitude Modulation)";
+  }
+
+  identity DP-QAM8 {
+    base modulation;
+    description
+      "DP-QAM8 (Dual Polarization Quadrature Amplitude Modulation)";
+  }
+
+  identity DC-DP-QAM8 {
+    base modulation;
+    description
+    "DC DP-QAM8 (Dual Polarization Quadrature Amplitude Modulation)";
+  }
+
+  identity DP-QAM16 {
+    base modulation;
+    description
+      "DP-QAM16 (Dual Polarization Quadrature Amplitude Modulation)";
+  }
+
+  identity DC-DP-QAM16 {
+    base modulation;
+    description
+    "DC DP-QAM16 (Dual Polarization Quadrature 
+    Amplitude Modulation)";
+  } 
+

```

Comment:
What does DC stand for ? need to update the description

Reply:
@ggalimba56 : the meaning is Dual Carrier. Addressed in the code.